docs/context.md

```diff
@@ -66,6 +66,8 @@ Our end goal is to provide contextual access to any of the properties available
 [Outreach API](https://api.outreach.io/api/v2/docs#prospect), but at the moment, we support in
 [SDK](../src/context/keys/ProspectContextKeys.ts) next properties:
 
+- **pro.accnme** Prospect name.
```

Comment:
It is "Prospect account name" - not :"Prospect name"

Reply:
it's fixed